pom.xml

```diff
@@ -138,13 +138,15 @@
 			<artifactId>datatables-servlet2</artifactId>
 			<version>0.8.11</version>
 		</dependency>
+		<!-- Added due to the usage of freemarker.template.LocalizedString -->
 		<dependency>
-			<groupId>org.fenixedu</groupId>
-			<artifactId>fenixedu-commons</artifactId>
+			<groupId>org.freemarker</groupId>
+			<artifactId>freemarker</artifactId>
+			<version>2.3.20</version>
 		</dependency>
 		<dependency>
-			<groupId>com.qubit.terra</groupId>
-			<artifactId>qub-docs</artifactId>
+			<groupId>org.fenixedu</groupId>
+			<artifactId>fenixedu-commons</artifactId>
```

Comment:
@joaofreitas-qubit why are we adding fenixedu-commons?

Reply:
Actually the dependency was already there, it seems that it just switched places with the freemarker one